### DISCUSSION TITLE
Are all components available as CSS components?

### DISCUSSION THREAD

**Discussion by ReneZYY** [emdgroup-liquid/liquid#414] [Q&A] [answered]
Hi,

I am working in Merck LS-Q. Trying to utilize this great library to our Wiki site. 

We can only use CSS way, Web-component is not possible in our scenario.

But I can only see ld-button.css, what if I need other components' CSS like sidenav, I didn't find them, thanks.

**Comment by renet** [comment, marked as answer]
According to the [docs](https://liquid.emd.design/liquid/introduction/getting-started/), you can either import `@emdgroup-liquid/liquid/dist/css/liquid.css` from the npm package, which includes all the base styles and every available CSS component at once, or you can import `@emdgroup-liquid/liquid/dist/css/liquid.global.css` for only the base styles and then import the styles for the desired CSS component from `@emdgroup-liquid/liquid/dist/css/<component-name>.css`.

If you can't use a Node.js package manager like npm or yarn, you may also download the package directly as `.tgz` file from: `https://registry.npmjs.org/@emdgroup-liquid/liquid/-/liquid-<version-number>.tgz`, so for example the currently latest version can be downloaded here: https://registry.npmjs.org/@emdgroup-liquid/liquid/-/liquid-2.4.3.tgz

Please be aware that only components that have a CSS component version available (if that's the case, the docs will show a CSS component example), can be imported using these methods. The `ld-sidenav` you mentioned, for example, is not available as CSS component, because it's not usable without certain JavaScript logic that is encapsulated in the web component.

**Comment by borisdiakur** [comment]
Seems this issue is really just a question and hasn't been active in a while. So I'm converting it to a Q&A discussion.
